scripts/count_gene_mutations.R

```diff
@@ -0,0 +1,154 @@
+#!/usr/bin/env Rscript
+
+# Count the number of samples mutated for each gene in a MAF file.
+
+# This script reads in a MAF file and writes out a table of the genes with
+# mutations. The table includes the number of samples that have at least one 
+# mutation in the gene, as well as the total number of mutations detected across
+# all samples.
+
+# Option descriptions:
+#
+# --maf :  File path to MAF file to be analyzed. Can be .gz compressed.
+# --outfile : The name of the output file to create
+# --vaf: Minimum variant allele fraction of mutations to include.
+# --min_depth: Minimum sequencing depth to call mutations.
+# --include_syn: Flag to include synonymous mutations in counts.
+
+# Example invocation:
+#
+# Rscript count_gene_mutations.R \
```

Comment:
Maybe also show a 1-line version of this? I am not suggesting to delete this, but to complement with a version `\`-free since separating onto new lines may be new for them. 
That said, we probably want them to get in the habit of writing clearly across multiple lines, so I don't want to add a one-liner and lose that message.